Sources/Testing/ABI/ABI.swift

```diff
@@ -125,6 +126,14 @@ extension ABI {
       VersionNumber(6, 3)
     }
   }
+
+  /// A namespace and type representing the ABI version whose symbols are
+  /// considered experimental.
+  enum ExperimentalVersion: Sendable, Version {
+    static var versionNumber: VersionNumber {
+      VersionNumber(99, 99)
```

Comment:
Nitpick: the stdlib would write this as just 99.0, not 99.99, right?

Reply:
I wasn't sure, but I'm fine with that